tests/tlstest.py

```diff
@@ -2393,6 +2621,29 @@ def connect():
 
     test_no += 1
 
+    print("Test {0} - good mutual X.509 Ed25519, PHA, TLSv1.3".format(test_no))
```

Comment:
We dont have this (anything) covered with ed448 client certs. Is is needed?

Reply:
no, I have both ed25519 and ed448 for server certificates as technically those are different key _types_ in certificates, so need to verify that both branches work as expected. For client certs we're reusing that code, so as long as one EdDSA works, other should too